sphinx/texinputs/sphinx.sty

```diff
@@ -413,6 +414,22 @@
 \newcommand\sphinxsetup[1]{\setkeys{sphinx}{#1}}
 
 
+%% ALPHANUMERIC LIST ITEMS
+\newcommand\sphinxsetlistlabels[5]
```

Comment:
I decided not to mention this macro in the documentation. But advanced LaTeX users can redefine it to their liking to customize labels of enumerated lists even without using any package.